frontend/catalyst/preprocess.py

```diff
@@ -64,6 +168,18 @@ def null_postprocessing(results):
 
 def catalyst_acceptance(op: qml.operation.Operator, operations) -> bool:
     """Specify whether or not an Operator is supported."""
+    if isinstance(
+        op,
+        (
+            catalyst.pennylane_extensions.Adjoint,
+            catalyst.pennylane_extensions.MidCircuitMeasure,
+            catalyst.pennylane_extensions.ForLoop,
+            catalyst.pennylane_extensions.WhileLoop,
+            catalyst.pennylane_extensions.Cond,
+        ),
+    ):
```

Comment:
```suggestion
    elif isinstance(op, catalyst.jax_tracer.HybridOp):
```

I think the above should work.

Reply:
No it does not unfortunately because QCtrl handle decomposition differently by defining `.decomposition()`

Reply:
I think what we should do here is have a standard property where nested tapes are stored. Any op with that property can then be recursed into (assuming it wasn't decomposed first).